src/base.cc

```diff
@@ -67,14 +67,14 @@ ConstArray2D arrayFromVector(std::vector<double> const &vec, int nAxes) {
 }
 
 Array2D arrayFromVector(std::vector<double> &vec, int nAxes) {
-    if (vec.size() % nAxes != 0) {
+    int nPoints = vec.size() / nAxes;
+    if (nPoints * nAxes != vec.size()) {
```

Comment:
I don't understand this change. What's wrong with the original test?

Reply:
I felt the test was more obvious the new way. I'd seen it done elsewhere in our stack and I preferred it.